<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBuild</class>
 <widget class="QWidget" name="AddBuild">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/images/ALogo.png</normaloff>:/images/images/ALogo.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButtonChooseDirectory">
     <property name="text">
      <string>Choose...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Build Number</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEditBuildID"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Directory</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditDirectory"/>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEditDescription"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEditName"/>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditDirectory</tabstop>
  <tabstop>pushButtonChooseDirectory</tabstop>
  <tabstop>lineEditBuildID</tabstop>
  <tabstop>lineEditName</tabstop>
  <tabstop>lineEditDescription</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>AddBuild</receiver>
   <slot>cancelClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>AddBuild</receiver>
   <slot>okClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonChooseDirectory</sender>
   <signal>clicked()</signal>
   <receiver>AddBuild</receiver>
   <slot>chooseClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>cancelClick()</slot>
  <slot>okClick()</slot>
  <slot>chooseClick()</slot>
 </slots>
</ui>
